src/index.js

```diff
@@ -64,13 +64,18 @@ export default class TimeAgo extends Component<DefaultProps, Props, void> {
   timeoutId: ?number;
   isStillMounted: boolean = false;
 
+  // Use Number.isNaN if available, otherwise use polyfill (not available in some browsers inc. IE11)
+  checkIsNaN = (value) => {
+    return (Number.isNaN !== undefined) ? Number.isNaN(value) : typeof value === 'number' && isNaN(value);
+  }
```

Comment:
This function is not directly related to the component (as it doesn't use `this`), perhaps extract it outside of the component to make it more like an utility function? What do you think about that :wink: